src/pages/ReviewResults.js

```diff
@@ -13,6 +13,7 @@ const SECTION = {
 
 export default function ReviewResults(props) {
   const darId = props.match.params.referenceId;
+  const status = props.match.params.referenceId;
```

Comment:
This looks like a copy paste error. Shouldn't it be looking for the `status` param?